marketplace/netlify-build/extension.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "netlify-button",
+  "name": "Netlify Build and Preview",
+  "src": "http://localhost:1234",
+  "sidebar": true,
+  "parameters": {
+    "installation": [
+      {
+        "id": "netlifySiteIds",
+        "type": "Symbol",
+        "name": "Comma seperated Netlify Site Id list (max 3)",
+        "description": "Example: 14234e3,143248,35233",
```

Comment:
Provide values that resemble Netlify site IDs

Reply:
Done.